NEXT_CHANGES.rst

```diff
@@ -0,0 +1,102 @@
+Instructions
+============
+
+Use this as a staging ground for CHANGES.rst. In other words, describe the
+changes and additions to ixdat's API associated with your contribution. The idea is
+that what you write here informs other developers what is on its way and then will be
+copied to CHANGES.rst when the next version of ixdat is distributed. Please include
+links to relevant Issues, Discussions, and PR's on github.
+
+Coming changes for ixdat 0.2.0
+==============================
+
+API Changes
+-----------
+
+ixdat.measurement
+^^^^^^^^^^^^^^^^^
+
+- Generalization of multiple calibrations:
+
+  ``Measurement.calibration`` is deprecated in favor of ``Measurement.calibration_list``.
+  Use ``add_calibration()``. Setting ``calibration`` is deprecated.
+
+
+ixdat.techniques
+^^^^^^^^^^^^^^^^
+
+- Renamed measurement technique class: ``CyclicVoltammogram`` (replaces ``CyclicVoltammagram``)
+  The old name is deprecated.
+
+- Renamed properties in ``ECMeasurement`` and inheriting classes:
+
+  - ``v_name`` (replaces ``V_str``)
+  - ``j_name`` (replaces ``J_str``)
+  - ``E_name`` (replaces ``E_str``)
```

Comment:
For code that should represent something a scientist could conceivably write on paper or on the blackboard, I want `ixdat` to always stay as close as possible to what they'd write. I'd never write "e" for potential or "i" for current (I have seen the latter though). I guess "E" and "I" carry over from the biologic names for the raw data. Calling normalized current "j"  (lowercase) is pretty standard, I think. "v" for calibrated potential might be less standardized (E, V, and U are maybe more common) but capital "V" clashes with the symbol for the unit Volt. Hm... now I kind of regret not calling it "U". what do you think? It could be "U" and "J" for calibrated potential and current, to keep them all uppercase. What do you think? Now's probably the last chance to change them, so it should be something we agree on.

Reply:
I like "U" much more than "v". Didn't you use "U" also in the figure captions in EC_MS?

Reply:
oh, and yes, I guess upper case "J" would make more sense for consistency. though the lower case letter is what one would use for current density on the blackbord, isn't it?

Reply:
Yeah, typically ``v_name`` is "U vs RHE / [V]"   (same in EC_MS though it was called ``V_str``)
So... yeah lets do "J_name" and "U_name" then. Tribalism for the win. Real electrochemists call potential "U", haha ;) 

On it. Bleh, more refactoring.

Reply:
was just reminded when opening ec.py to start that ``j_name`` is typically "J / [mA cm$^{-2}$]", haha. 
"E", "I", "U", and "J" are then least confusing.

Reply:
Oh, but there's more:
``ec_meas.v`` and ``ec_meas.j`` returns the potential and current. Should that be ``ec_meas.U`` and ``ec_meas.J`` ?
And aren't cyclic voltammograms and linear sweeps usually plotted as "J-V curves"?

Reply:
Yes, I guess it should be `ec_meas.U` and `ec.meas.J`, even though when writing with this syntax it feels more like code and less like electrochemistry, so the capitalization seems a bit off. and yes, they are called J-V curves, but I with `U_name` being "U vs RHE / [V]" I think having the U there is the least confusing, as it draws attention to the naming convention in the plot. if someone doesn't like the U there, they are warned and can change it to V manually.